packages/server/lib/api.coffee

```diff
@@ -238,31 +273,6 @@ module.exports = {
     .timeout(timeout)
     .catch(tagError)
 
-  createSignin: (code) ->
-    machineId()
```

Comment:
this needs to be reimplemented in the new logic - and the API needs to ensure this property is being handled.

Reply:
b8e47468d0b2a56de38bec04e4bc857fbccb4628 adds it to query params that we send to the dashboard